testing/ruuvi-prometheus/alpine/APKBUILD

```diff
@@ -0,0 +1,35 @@
+# Contributor: Joonas Kuorilehto
+# Maintainer:
+pkgname=ruuvi-prometheus
+pkgver=0.1
+pkgrel=0
+pkgdesc="Ruuvi sensor Prometheus exporter"
+url="https://github.com/joneskoo/ruuvi-prometheus"
+arch="all"
+depends="bluez"
+license="BSD-2-Clause"
+options="!check"  # No test suite
+depends=""
+makedepends="go"
+install=""
+subpackages="$pkgname-openrc"
+
+source="$pkgname-$pkgver.tar.gz::https://github.com/joneskoo/ruuvi-prometheus/archive/v$pkgver.tar.gz
+        ruuvi-prometheus.initd
+        "
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+	cd "$builddir"
```

Comment:
not required, this is done automatically

Reply:
I think this was from the template generator I used following the docs. I didn't add it. Will clean.